cups/ipp.c

```diff
@@ -4701,8 +4689,8 @@ ippValidateAttribute(
 		userpass[256],		/* Username/password from URI */
 		hostname[256],		/* Hostname from URI */
 		resource[1024];		/* Resource from URI */
-  int		port,			/* Port number from URI */
-		uri_status;		/* URI separation status */
+  int		port;			/* Port number from URI */
+  http_uri_status_t		uri_status;		/* URI separation status */
```

Comment:
Fix indentation.

Reply:
Indentation matches the other text, even if it is rendered funny in GitHub